Description: Downgrade bit-vec dependency
Author: Mike Hommey <glandium@debian.org>
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
Index: git-cinnabar/Cargo.toml
===================================================================
--- git-cinnabar.orig/Cargo.toml
+++ git-cinnabar/Cargo.toml
@@ -58,7 +58,7 @@ version = "2.0.1"
 version = "0.3"
 
 [dependencies.bit-vec]
-version = "0.8"
+version = "0.6"
 
 [dependencies.bitflags]
 version = "2"
